github.com.js

```diff
@@ -17,6 +17,11 @@ function injectHtml() {
         '<label><input type="checkbox" class="js-collapse-deletions" checked="yes">-</label>' +
     '</span>').insertAfter('.actions, .file-actions');
 
+  $('<span class="pretty-pull-requests hide-whitespace">' +
+        '<label>Hide whitespace changes <input type="checkbox" class="js-hide-whitespace"></label>' +
+    '</span>').insertAfter('.diffbar-item.diffstat');
+  $('.js-hide-whitespace').prop('checked', isWhitespaceHidden('w'));
```

Comment:
What is the 'w' that is passed to isWhitespaceHidden? It looks like maybe you used to have a param in that function, and now don't?

Reply:
Good call, thanks! I have fixed this now.